libs/cli/langgraph_cli/config.py

```diff
@@ -485,29 +619,31 @@ def python_config_to_docker(config_path: pathlib.Path, config: Config, base_imag
             ],
         )
     )
-    store_config = config.get("store")
-    env_additional_config = (
-        ""
-        if not store_config
-        else f"""
-ENV LANGGRAPH_STORE='{json.dumps(store_config)}'
-"""
-    )
-    if (auth_config := config.get("auth")) is not None:
-        env_additional_config += f"""
-ENV LANGGRAPH_AUTH='{json.dumps(auth_config)}'
-"""
-    return f"""FROM {base_image}:{config['python_version']}
-
-{os.linesep.join(config["dockerfile_lines"])}
 
-{installs}
+    env_vars = []
```

Comment:
refactor -- the previous version of the code was too mind boggling to understand

Reply:
nice, much more readable